### PR TITLE
Feature 2024 01 01 preview impl

### PR DESCRIPTION
**Pull request by jamesfan1**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -719,10 +719,10 @@
 # ServiceOwners:                                                   @privlinksuppgithub
 
 # PRLabel: %Nginx
-/sdk/nginx/                                                        @spencerofwiti @bangbingsyb
+/sdk/nginx/                                                        @spencerofwiti @bangbingsyb @jamesfan
```
Comment (jsquire) [inline .github/CODEOWNERS:722]: Given that this is a management library, we should move the nginx block down to the management section around L951 and follow the pattern there to restrict to the Resource Manager path.

Please be sure to maintain alphabetical ordering of label names when moving.

**Review by jsquire — DISMISSED** [review]
@jamesfan1:

Your account lacks the required public GitHub organizations and permissions required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

**Comment by jsquire** [comment]
How is this different from #43462, and do we need both of these?

**Comment by jamesfan1** [comment]
hi @jsquire thank you for reviewing, we just need this one, I will close the last PR

**Comment by jamesfan1** [comment]
I have fixed conflict and just had the Azure SDK Partners permission added

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -956,6 +950,13 @@
 # ServiceLabel: %Network - Front Door %Mgmt
 # ServiceOwners:                                                   @ptnan7
 
+# PRLabel: %Nginx
+/sdk/nginx/                                                        @jamesfan1 @bangbingsyb @spencerofwiti
```
Comment (jsquire) [inline .github/CODEOWNERS:954]:
```suggestion
/sdk/nginx/Azure.ResourceManager.*/                                @jamesfan1 @bangbingsyb @spencerofwiti
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -956,6 +950,13 @@
...
+
+# ServiceLabel: %Nginx
```
Comment (jsquire) [inline .github/CODEOWNERS:956]:
```suggestion
# ServiceLabel: %Nginx %Mgmt
```

**Comment by jsquire** [comment]
@jamesfan1:  Account permissions and memberships look good.   I made two small corrections to CODEOWNERS to ensure that it follows the expected pattern.   Going forward, you'll need to work with @ArthurMa1978 and team on the review for the library.

**Comment by ArthurMa1978** [comment]
Hey @jamesfan1 please re-record all tests and update the records.

**Comment by jamesfan1** [comment]
@ArthurMa1978 

facing some issue when pushing the recording

Unhandled exception: Azure.Sdk.Tools.TestProxy.Common.Exceptions.HttpException: Invocation of "git push origin net/nginx/Azure.ResourceManager.Nginx_424be0482a" had a non-zero exit code -1.
StdOut:
StdErr: remote: The 'Azure' organization has enabled or enforced SAML SSO.
remote: To access this repository, you must re-authorize the OAuth Application 'GitHub CLI'.
fatal: unable to access 'https://github.com/Azure/azure-sdk-assets/': The requested URL returned error: 403

**Comment by jamesfan1** [comment]
I have uploaded the recording, please check when available

**Comment by jamesfan1** [comment]
@ArthurMa1978  all test passed, please help review and approve. Thank you

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**